packages/bsky/src/index.ts

```diff
@@ -116,6 +117,12 @@ export class BskyAppView {
       adminPasses: config.adminPasswords,
     })
 
+    if (config.statsigKey) {
+      Statsig.initialize(config.statsigKey, {
+        environment: { tier: config.statsigEnv },
+      })
```

Comment:
[This is async](https://docs.statsig.com/server/nodejsServerSDK#initialize-the-sdk), but this happens on server start and then periodically in the background to get updated gates, so I'm thinking that we probably don't need to await here.

Reply:
Confirmed with StatSig team: calls to `checkGateSync` (checking a feature flag) that happen synchronously after this call will be fine and return `false` by default. However, gate check calls should NOT happen before `initialize()` is called, since they will throw.

Since this is called on server startup, any gate check calls should happen after `initialize()`, so I think we don't need to worry about this. Could use a second opinion though.